### ISSUE TITLE
Source Code Formatting (XML): comments not shown.

### ISSUE THREAD

**Issue by uropro** [Requarks/wiki#1600]
**Describe the bug**
XML comments are not shown withing source code blocks:
```
```xml
<!-- this comment is not shown
-->
\```
```
( without the "\" )

**To Reproduce**
1. Go to create a new page markdown
2. Paste XML comment within ```
3. The pre-rendering looks fine.
3. Create page. 
Note: Behaviour is not dependent on "xml" tag, also happens with "diff".
Similar to bug #1172 

**Expected behavior**
hard to demonstrate, because it does not work here on github either.
``` <!-- this comment is shown --> 
```

**Host Info (please complete the following information):**
 - OS: FreeBSD 12.1
 - Wiki.js version: 2.1.113
 - Database engine: PostgresSQL 11.7

**Comment by uropro** [comment]
A work-around is to replace the opening `<` of the comment by `&lt;`. The pre-rendering will not convert the `&lt;` to the `<` character, but the final rendering does.
So enter this, (omit the back-slash at the closing end):
```
```xml
&lt;!-- this comment is now shown correctly in final rendering
-->
\```
```

**Comment by NGPixel** [comment]
Fixed by b529ad21c97a7093e9ecc35e2e72fb8148f0c9d0
